csseries/cookit
# WIP: Train tf lite object-detection model

- Create train, val, test database 
- Train model conveniently with command line interface and basic parameters
- Convert fiftyoneImageDetection to expected format by tf make_model
- upload / download files and dataset to cloud
- create (balanced) random slices
- and many more

To do:
- import of trained model and class labels more conveniently
- make the order of scores, bboxes and classes more reliable
